src/ui/components/Button/index.js

```diff
@@ -1,13 +1,31 @@
+/* @flow */
 import makeClassName from 'classnames';
 import { oneLine } from 'common-tags';
 import * as React from 'react';
-import PropTypes from 'prop-types';
 
 import Link from 'amo/components/Link';
 import log from 'core/logger';
 
 import './styles.scss';
 
+type Props = {|
+  buttonType: string | 'none',
```

Comment:
We should use a better type that is the intersection of all possible values of `BUTTON_TYPES`. This must be done manually though...

```
type ButtonType =
  | 'neutual'
  | 'action'
  | ...;
```

Reply:
Yes, it means there are other components to fix. See the diff I pasted above, that should help you (hopefully)!

Reply:
oh I see in your recommended patch how you we are importing this to each file and fixing this 👍